src/createServer.js

```diff
@@ -1,11 +1,233 @@
+/* eslint-disable function-paren-newline */
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const router = express.Router();
+const cors = require('cors');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+  const mockUsers = [];
+  const mockExpenses = [];
+
+  app.use(cors());
+  app.use(express.urlencoded({ extended: true }));
+  app.use(express.json());
+  app.use('/', router);
+
+  app.get('/users', (req, res) => {
+    if (mockUsers.length === 0) {
+      return res.send([]);
+    }
+    res.status(200).send(mockUsers);
+  });
+
+  app.post('/users', (req, res) => {
+    const { name } = req.body;
+
+    if (!name) {
+      res.status(400).send('Name is required!');
+
+      return;
```

Comment:
You can return the res itself, just move this _return_ to the beginning of line 29